bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/BOSWebServerManager.java

```diff
@@ -224,13 +229,18 @@ private void updateRuntimeLocationIfNeeded() {
             if (runtime instanceof org.eclipse.wst.server.core.internal.Runtime && runtime.getLocation() != null
                     && !runtime.getLocation().toFile().getAbsolutePath().equals(tomcatInstanceLocation)) {
                 final IRuntimeWorkingCopy copy = runtime.createWorkingCopy();
-                final String oldLocaiton = copy.getLocation().toFile()
+                final String oldLocation = copy.getLocation().toFile()
                         .getAbsolutePath();
                 copy.setLocation(Path.fromOSString(tomcatInstanceLocation));
                 final File serverXmlFile = new File(tomcatInstanceLocation, "conf" + File.separatorChar + "server.xml");
-                // for Windows, we need to escape \
-                FileUtil.replaceStringInFile(serverXmlFile, oldLocaiton,
-                        tomcatInstanceLocation.replaceAll("\\\\", "\\\\\\\\"));
+                try {
+                    com.google.common.io.Files.write(
```

Comment:
I don't see the dependance on com.google.io in the MANIFEST.MF. Can you check if you didn't forget to commit this file?